scripts/translation/add-missing-strings.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+for id in $(grep "trans-unit id=" /cvs/GALAXY/uyuni-fork/branding/java/code/src/com/redhat/rhn/branding/strings/StringResource_en_US.xml | sed 's/.*trans-unit id="\(.\+\)".*/\1/g'); do
```

Comment:
`/cvs/GALAXY/uyuni-fork/` sounds like a local prefix. Maybe cutting it off or passing it as an argument would be better.

Reply:
fixed